test/test_framework.py

```diff
@@ -1628,33 +1692,35 @@ def callback_method(self, event: ops.EventBase):
         self.called = True
 
 
-@patch('sys.stderr', new_callable=io.StringIO)
-class BreakpointTests(BaseTestCase):
+class TestBreakpoint:
 
-    def test_ignored(self, fake_stderr: io.StringIO):
+    def test_ignored(
+        self,
+        request: pytest.FixtureRequest,
+        caplog: pytest.LogCaptureFixture,
```

Comment:
In the other PR too: Let's standardise on using a trailing comma like this, to avoid multiple lines appearing in the diff if we add arguments.